docs/source/software_process.rst

```diff
@@ -36,12 +36,13 @@ codecov_, which shows line by line which lines are covered by the tests.
 Installation Requirements
 -------------------------
 
-:pynwb:`setup.py <blob/dev/setup.py>` contains a list of package dependencies and their version ranges allowed for
+pyproject.toml_ contains a list of package dependencies and their version ranges allowed for
```

Comment:
```suggestion
:pynwb:`pyproject.toml <blob/dev/pyproject.toml>` contains a list of package dependencies and their version ranges allowed for
```

Reply:
You could use a reference since it's an internal link instead of the full http address. That's how the other pynwb links are structured but not sure how important it is.

Reply:
Both work. My way just mirrors hdmf.